content/scout/_index.md

```diff
@@ -47,19 +47,14 @@ grid:
     icon: upgrade
 ---
 
-Container images are often built from layers of other container images and
-software packages. These layers and packages can contain vulnerabilities that
-make your containers and the applications they run vulnerable to attack.
+Container images consist of layers and software packages, which are susceptible to vulnerabilities.
+These vulnerabilities can compromise the security of containers and applications.
 
-Docker Scout can proactively help you find and fix these vulnerabilities,
-helping you create a more secure software supply chain. It does this by analyzing your images and creating a full inventory of the
-packages and layers called a [Software bill of materials (SBOM)](https://ntia.gov/sites/default/files/publications/sbom_at_a_glance_apr2021_0.pdf).
-It then correlates this inventory with a continuously updated vulnerability
-database to identify vulnerabilities in your images.
+Docker Scout is a solution for proactively enhancing your software supply chain security.
+By analyzing your images, Docker Scout compiles an inventory of components, also known as a Software Bill of Materials (SBOM).
+The SBOM is matched against a continuously updated vulnerability database to pinpoint security weaknesses.
 
-You can use Docker Scout in Docker Desktop, Docker Hub, the Docker CLI, and in
-the [Docker Scout Dashboard](./dashboard.md). Docker Scout also supports
-integrations with third-party systems, refer to [Integrating Docker
-Scout](./integrations/index.md) for more information.
+You can use Docker Scout in Docker Desktop, Docker Hub, the Docker CLI, and the Docker Scout Dashboard.
```

Comment:
Is there a quick way to indicate the Docker Scout is a separate product / service / platform in itself, and that you can interact with it via these UIs?